pset-functions.js

```diff
@@ -31,6 +31,42 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+function blackJack(playerCardScore, dealerCardScore){
+  // both over 21
+  if (playerCardScore > 21 && dealerCardScore > 21){
+    return 0
+  }
+  // Draw case
+  else if (playerCardScore===dealerCardScore){
+    return playerCardScore
+  }
+
```

Comment:
Katie, really great use of if/else statements to capture every possible scenario in proper sequence! Be careful not to put line spacing between else/if statements that are chained together. Only provide that space if you are starting a completely new if/else statement.